camera_count : 1
frame_rate: 30    # 0: trigger mode

camera1:
    name: cam1
    serial_number: 23641283

